app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -61,6 +62,11 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
 
         val intent = SafeIntent(intent)
 
+        if (Settings.getInstance(this).shouldShowFirstrun() &&
+                !Performance.processIntentIfPerformanceTest(intent, this)){
+            components.appStore.dispatch(AppAction.ShowFirstRun)
```

Comment:
nitpick (blocking): we need to update the kdoc on `ShowFirstRun` to say it's not just for tests.